domain_model.md

```diff
@@ -0,0 +1,27 @@
+# Domain Models
+
+
+| **Class**      | Dish         |
+|----------------|--------------|
+| **Properties** | name, price  |
+| **Actions**    |               |
+
+| **Class**      | Menu         |
+|----------------|--------------|
+| **Properties** | dishes(list)  |
+| **Actions**    | see, add_dish |
+
+| **Class**      | Takeaway        |
+|----------------|--------------|
+| **Properties** |   |
+| **Actions**    | see_menu, choose, exact_total, place_order |
+
+| **Class**      | Order         |
+|----------------|--------------|
+| **Properties** | order(list), quantities|
+| **Actions**    | add, see, total_check, send_text|
+
+| **Class**      | Text         |
+|----------------|--------------|
+| **Properties** | message  |
+| **Actions**    | send_message |
```

Comment:
This is a nice addition to your project as it really helps with visualizing what you were going for.